Lesson6.ipynb

```diff
@@ -27,7 +34,686 @@
+    "        self.isCaptain = \"Yes\" if isCaptain else \"No\"       #assign \"Yes\" if the isCaptian value is True else assigns \"No\"\n",
+    "        self.totalStepCount = totalStepCount\n",
+    "        Member.totalMembers += 1                            #totalMembers keeps increases by 1, everytime a new Member object is created"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The variable *`size`* is a class variable whose value is shared among all instances of this class. This can be accessed as *`Member.size`* from inside or outside the class.\n",
+    "\n",
+    "The first method ***`__init__()`*** is a special method, which is called class constructor or initialization method that Python calls when you create a new instance of this class.\n",
+    "\n",
+    "You declare other class methods like normal functions with the exception that the first argument to each method is ***`self`***. Python adds the ***`self`*** argument to the list for you; you do not need to include it when you call the methods.\n",
+    "\n",
+    "### Creating object instances of a class\n",
+    "\n",
+    "To create instances of a class, you call the class using class name and pass in whatever arguments its __init__ method accepts.\n",
+    "\n",
+    "*`object_name = Class_Name(args...) `*\n",
+    "\n",
+    "For example,\n",
+    "\n",
+    "*`teamMember = Member(\"John\", \"18\", \"Safire\", True, 500000)`*\n",
+    "\n",
+    "Once, the object has been created. You can access the object attributes using the dot operator. like,\n",
+    "\n",
+    "*`teamMember.name` (or) `getattr(teamMember, \"name\")`*\n",
+    "\n",
+    "***`getattr(self, name)`*** is a builtin method, should return the (computed) attribute value or raise an AttributeError exception if the attribute not available.\n",
+    "\n",
+    "And Class variable would be accessed using the class name. like,\n",
+    "\n",
+    "*`Member.size`* \n",
+    "\n",
+    "Now, let's try by putting everything together:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Member:\n",
+    "    'Holds the details of a team member'                    #documentation string\n",
+    "    totalMembers = 0                                        #Class variable\n",
+    "\n",
+    "    def __init__(self, name, age, teamName, isCaptain, totalStepCount):    #initialize method called when class object is created\n",
+    "        self.name = name\n",
+    "        self.age = age\n",
+    "        self.teamName = teamName\n",
+    "        self.isCaptain = \"Yes\" if isCaptain else \"No\"       #assign \"Yes\" if the isCaptian value is True else assigns \"No\"\n",
+    "        self.totalStepCount = totalStepCount\n",
+    "        Member.totalMembers += 1                            #totalMembers keeps increases by 1, everytime a new Member object is created\n",
```

Comment:
"totalMembers keeps increases by 1, everytime a new Member object is created"
Should be read
"totalMembers is increased by 1 every time a new instance of Member is created"